lib/WebService/Async/Onfido.pm

```diff
@@ -1034,6 +1033,52 @@ sub countries_list {
     });
 }
 
+=head2 supported_documents_list
+
+Returns an array of hashes of supported_documents for each country
+
+=cut
+
+sub supported_documents_list {
+    my $path = Path::Tiny::path(__DIR__)->parent(3)->child('share/supported_documents.json');
+    $path = Path::Tiny::path(
+        File::ShareDir::dist_file(
+            'WebService-Async-Onfido',
+            'supported_documents.json'
+        )
+    ) unless $path->exists;
+    my $supported_documents = decode_json_text($path->slurp_utf8);
```

Comment:
Can we cache this result ?

Reply:
the json file is actually added in `bom-config`, this is just for public usage

Reply:
Sorry I can't get your idea.
I mean, can I do like this ?
```state $supported_documents = decode_json_text($path->slurp_utf8)```